cmd/cored/main.go

```diff
@@ -177,11 +177,11 @@ func runServer() {
 		h = launchConfiguredCore(ctx, db, conf, processID)
 	} else {
 		chainlog.Messagef(ctx, "Launching as unconfigured Core.")
-		h = &core.API{
+		h = core.Handler(&core.API{
```

Comment:
The fact that `launchConfiguredCore` also contains a call to `core.Handler` caused a momentary hiccup in my understanding of the flow here. It might be a little clearer if this `else` branch called a parallel function, `launchUnconfiguredCore`, rather than calling `core.Handler` directly. A non-PR-blocking suggestion.